packages/commons/src/big-query/liquidity-indices-table/pull-data/pullClosestIndices.ts

```diff
@@ -0,0 +1,54 @@
+import { getBigQuery } from '../../client';
+import { getTableFullName } from '../../utils/getTableName';
+import { TableType } from '../../types';
+import { Address } from '../../../utils/convertLowercase';
+
+export type LiquidityIndexEntry = {
+    chainId: number;
+    rateOracle: Address,
+    liquidityIndex: number,
+    blockTimestamp: number
+}
+
+const mapToLiquidityIndices = (row: any) : LiquidityIndexEntry => row;
+
+export const pullClosestIndices = async (
```

Comment:
I'd rather export `getLiquidityIndexAt` in `getLiquidityIndexAt.ts` file and add this functionality in that file as well. `pullClosestIndicies` does not seem to have its standalone functionality (seems to be coupled only to `getLiquidityIndexAt`).

Reply:
I moved `getLiquidityIndexAt()` and `pullClosestIndicies()` in the same file named `getLiquidityIndexAt` at `packages/commons/src/big-query/liquidity-indices-table/pull-data/getLiquidityIndexAt.ts` [e0c8edf](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/34/commits/e0c8edf872ac0156572fc067bdb129d70fc36e1d)